src/Plugin.php

```diff
@@ -99,30 +99,31 @@ function on_save_post($post_id) {
 				return;
 			}
 
-			$shlink = $this->get_post_shlink($post);
-			$site_url = parse_url(get_site_url());
-
 			$request = [
 				'longUrl' => $long_url,
-				'title' => $post->post_title,
-				'tags' => [
+				'title' => $post->post_title
+			];
+
+			$site_url = parse_url(get_site_url());
+			if (! empty($site_url['host'])) {
```

Comment:
Can you add a comment here explaining why this might be empty, and the fact it shouldn't actually create one with no tags but instead just update an existing one that already has tags?

Reply:
Yep, will do.